### DISCUSSION TITLE
How do you seed data in migrations using ActiveModels?

### DISCUSSION THREAD

**Discussion by Drakota** [SeaQL/sea-orm#519] [Q&A] [answered]
I'm trying to seed data directly inside a migration, but I'm unable to get a DatabaseConnection handle to be able to do it.
```rust
let db = manager.get_connection();

let board = board::ActiveModel {
    name: Set("Test Board".to_owned()),
    ..Default::default()
};

board.save(db).await?;
```
I tried something like this, but I'm getting:
`the size for values of type 'dyn sea_schema::migration::sea_orm::ConnectionTrait' cannot be known at compilation time the trait 'std::marker::Sized' is not implemented for 'dyn sea_schema::migration::sea_orm::ConnectionTrait'`.

**Comment by billy1624** [comment, marked as answer]
Hey @Drakota, thanks for reporting this. I think we can relax the constraint and make it work.

Please check
- #522
- #523

**Comment by Drakota** [comment]
That looks great! Thank you for your work on sea-orm.

**Reply by billy1624** [reply]
Thanks! Do share SeaORM with your friends :P
